bedrock-devnet/devnet/__init__.py

```diff
@@ -294,6 +289,20 @@ def convert_anvil_dump(dump):
 
     return dump
 
+def convert_forge_dump(dump):
```

Comment:
Are you sure this is necessary? I don't think it is if you look at the geth genesis json marshalling

Reply:
@tynes I removed this function, which means that the generated allocs file now uses hexadecimals for account balances instead of ints. I could not get

```go
balance, ok := new(big.Int).SetString(account.Balance, 16)
```

to work for some reason I can't figure out. I tried hardcoding "0x0" or "0x1" for `account.Balance` and it always fails. I've changed this code to be

```go
balance, ok := new(big.Int).SetString(strings.TrimPrefix(account.Balance, "0x"), 16)
```

as that the only way it correctly parses. The alternative to this would be to parse account balances as int before saving the allocs file (as was being done via `convert_forge_dump` function

Reply:
Refactored to:

```go
var balance math.HexOrDecimal256
err := balance.UnmarshalText([]byte(account.Balance))
if err != nil {
	return nil, fmt.Errorf("failed to parse balance for %s: %w", address, err)
}
memDB.AddBalance(address, (*big.Int)(&balance))
```